### DISCUSSION TITLE
This site can’t be reached:localhost refused to connect.

### DISCUSSION THREAD

**Discussion by CWi11iams** [frangoteam/FUXA#725] [General]
Greetings! I am trying to get FUXA running on my Raspberry Pi . I got the software installed but when I go to connect to the http://localhost:1881/ in chromium I just get the "this site can't be reached" message. Is anyone else having this issue? I am not sure why I am getting it. 
Thank you!
Chase

**Comment by unocelli** [comment]
Hi,
did you start the server?
`cd ./server`
`npm start`
